auditbeat/module/auditd/audit_linux.go

```diff
@@ -568,7 +568,7 @@ func addProcess(p aucoalesce.Process, m common.MapStr) {
 		process["name"] = p.Name
 	}
 	if p.Exe != "" {
-		process["exe"] = p.Exe
+		process["executable"] = p.Exe
```

Comment:
This package has a data.json and I think an execve.json that should be updated. You can run `go test . -data` on Linux to update them.

Reply:
Done, it unfortunately also removed some entries which are probably available in other envs.